view/frontend/web/css/source/_module.less

```diff
@@ -298,6 +298,10 @@
         pointer-events: none;
     }
 
+    .ap-promotional-message {
+        padding-top: 1rem;
+        padding-bottom: 1rem;
```

Comment:
It looks like the rest of this file is using `px` so we should probably be consistent with that.